qcodes/instrument_drivers/QuTech/IVVI.py

```diff
@@ -114,10 +114,7 @@ def __init__(self, name, address, reset=False, numdacs=16, dac_step=10,
                            label='Dac voltages',
                            get_cmd=self._get_dacs)
 
-        self.add_function(
-            'trigger',
-            call_cmd=self._send_trigger
-        )
+        self.trigger = self._send_trigger
```

Comment:
isn't it better to do it on the class level? say, just add `trigger = _send_trigger` after `def _send_trigger(..):.....`?

Reply:
in our context, indeed, that doesn't add much value. With my suggestion, `trigger` method would've been visible on the class level, as in without the need for instantiating an instance to see that such a method exists. I guess, it's just probably a "better" programming practice.